### PR TITLE
[TASK] Change superType to dictionary

### PR DESCRIPTION
**Pull request by gerdemann**

### CODE REVIEW

**Comment by bwaidelich** [comment]
:+1: good catch!

File: Configuration/NodeTypes.Profile.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 # A simple plugin that allows "Frontend users" to edit their profile
 #
 'Flowpack.Neos.FrontendLogin:Profile':
-  superTypes: ['TYPO3.Neos:Plugin']
+  superTypes: 
+  	'TYPO3.Neos:Plugin': TRUE
```
Comment (hlubek) [inline Configuration/NodeTypes.Profile.yaml]: The indentation looks wrong here.

Comment (gerdemann) [inline Configuration/NodeTypes.Profile.yaml]: This is now corrected. ;)

**Comment by hlubek** [comment]
:+1:

**Comment by aertmann** [comment]
remember to release a new version for this to be available..

**Pull request merged**